lib/Dialect/TritonGPU/Transforms/WarpSpecialization/AutomaticWarpSpecialization.cpp

```diff
@@ -57,4 +67,6 @@ void AutomaticWarpSpecialization::runOnOperation() {
   pm.addPass(createTritonGPUOptimizePartitionWarps());
   if (failed(runPipeline(pm, getOperation())))
     return signalPassFailure();
+  // clear attributes
+  clearAttributes(getOperation());
```

Comment:
passing kScheduledMaxStageAttrName into the pipeliner is intentional because it triggers TMA descriptor multibuffering....

I wouldn't mind a better suggestion on how to trigger this though :P